modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1579,6 +1661,31 @@ definition {
 		AppBuilderAdmin.validateIfSearchItemIsNotFound(itemName = "Apple");
 	}
 
+	@description = "Make sure the search result show if the column is available to be added or already inserted in the table view"
+	@ignore = "true"
+	@priority = "5"
+	test ValidateDisabledFieldInTableView {
+		property portal.acceptance = "false";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "Custom Object Title",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		AppBuilderAdmin.createFormViewAndTableView(
+			fieldType = "Text Field",
+			formViewName = "My Form View 1",
+			tableViewName = "My Table View 1");
```

Comment:
@tainguyen100 
this test is failing while performing this step. Probably related to "Text Field" change. Would you please take a look?